src/flows/pipes/Visualization/view.tsx

```diff
@@ -184,12 +233,27 @@ const Visualization: FC<PipeProp> = ({Context}) => {
 
     let triggeredErrorThresholdMessage = ''
 
-    for (let i = 0; i < columns['_field'].data.length - 1; i++) {
-      const field = columns['_field'].data[i]
+    const values: any = Object.values(columns).filter(
```

Comment:
sorry, above comment should have been for this. doesn't this get clogged on large data sets?